atm.js

```diff
@@ -1,21 +1,44 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-}
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
-}
-
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+$(function(){
+
+	var cbalance = parseFloat( $("#checking .balance").text().replace("$", "") );
+	var sbalance = parseFloat( $("#saving .balance").text().replace("$", "") );
+
+	$( "#checking .depositbtn" ).on( "click", function(){
+		var cInput = parseFloat( $("#checking .balanceinput").val().replace("$", "") );
+		cbalance = cInput + cbalance;
+		$("#checking .balance").text('$' + cbalance);
+	});
+
+	$( "#checking .withdrawbtn" ).on( "click", function(){
+		var cInput = parseFloat( $("#checking .balanceinput").val().replace("$", "") );
+		cbalance = cbalance - cInput;
+		if(cbalance < 0){
+			if(sbalance > 0){
```

Comment:
This is the cleanest code I've seen of any ATM assignment!!! EXCELLENT indentation